bin/cli.ts

```diff
@@ -10,12 +10,15 @@ import { remoteConfig } from '../src/remoteConfig';
 import { initGeoReader } from '../src/common/geo';
 
 async function run(positionals: string[]) {
-  await Promise.all([remoteConfig.init(), initGeoReader()]);
+  await remoteConfig.init();
 
   switch (positionals[0]) {
     case 'api':
       tracer('api').start();
       startMetrics('api');
+
+      await initGeoReader();
+
```

Comment:
Only needed for api and api-private for now, this should lower memory consumption, will validate after merge